.github/workflows/nodejs.yml

```diff
@@ -34,8 +34,25 @@ jobs:
     - name: Build the files
       working-directory: ./hello-world
       run: npm run build
-
-    
-      
-      
- 
+    - name: Clone a repo
+      run: |
+        git config --global user.email "sumangala028@gmail.com"
+        git config --global user.name "sumangala028"
+        
+        git clone https://github.com/sumangala028/sample-repo.git
+        
+        cd sample-repo
+        echo hello >> README.md
+        echo "" >> README.md
+        
+        git add --all
+        git commit -m "README updated
```

Comment:
is this correct?

Reply:
"README updated" - That apostrophe missing. Right?